server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -361,6 +367,15 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 			this.checkpointService,
 		);
 
+		// Emit api counter every certain interval.
+		setInterval(() => {
```

Comment:
This setInterval should be cleared when the lambda is closed. Otherwise, this will cause memory leaks

Reply:
Added in the close function.